avionics/envs/x86/include/sensors/thermocouple.h

```diff
@@ -10,17 +10,14 @@
 /*Constants------------------------------------------------------------*/
 
 class Thermocouple : public ISensor {
-  private:
-    static constexpr int THERMOCOUPLE_DATA_ARRAY_SIZE = 1;
-
   public:
     void initSensor() {}
     void readData() {}
-    uint8_t dataLength() { return THERMOCOUPLE_DATA_ARRAY_SIZE; }
+    uint8_t dataLength() { return THERMOCOUPLE_DATA_LENGTH; }
     float *getData() { return data; }
     SensorStatus getStatus() { return SensorStatus::NOMINAL; }
 
   private:
-    float data[THERMOCOUPLE_DATA_ARRAY_SIZE] = {
+    float data[THERMOCOUPLE_DATA_LENGTH] = {
```

Comment:
Are we not simulating the thermocouple?

Reply:
It's currently not being read and is somewhat deprecated as we dont really need it on our current rockets.

Just made a task for the future though: https://trello.com/c/X1K0bPdu/270-enable-mb-thermocouple